ui/search/nuxeo-results-view.js

```diff
@@ -0,0 +1,453 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class';
+import '@polymer/iron-flex-layout/iron-flex-layout.js';
+import '@polymer/iron-collapse/iron-collapse.js';
+import '@nuxeo/nuxeo-elements/nuxeo-page-provider.js';
+import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
+import './nuxeo-search-form-layout.js';
+import './nuxeo-search-results-layout.js';
+
+{
+  /**
+   * `nuxeo-results-view`
+   * @group Nuxeo UI
+   * @appliesMixin Nuxeo.I18nBehavior
+   * @element nuxeo-results-view
+   */
+  class ResultsView extends mixinBehaviors([I18nBehavior], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <style include="nuxeo-styles iron-flex iron-flex-alignment">
+          .ellipsis {
+            text-overflow: ellipsis;
+            overflow: hidden;
+            white-space: nowrap;
+            display: block;
+          }
+
+          .capitalize {
+            text-transform: capitalize;
+          }
+
+          .form {
+            @apply --paper-card;
+            padding: 0;
+          }
+
+          .header,
+          #collapse {
+            padding: 0 16px;
+          }
+
+          .buttons {
+            @apply --buttons-bar;
+          }
+
+          .hidden {
+            visibility: hidden;
+          }
+
+          .count {
+            @apply --layout-horizontal;
+            @apply --layout-center;
+            @apply --layout-center-justified;
+            background: var(--nuxeo-badge-background);
+            color: white;
```

Comment:
should we use a variable here? considering dark theme, would white be ok in that context?